LICENSE

```diff
@@ -1,13 +1,202 @@
-Copyright 2018 University of Washington
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+                                 Apache License
```

Comment:
This was autogenerated using `cookiecutter`; I realize now that this is a different license than what was here before, although the license in `setup.py` was stated to be Apache2 (that did not change.) 

tbh I'm not really sure what the right thing to do is here.